controllers/network-policy-controller/networkpolicycontroller.go

```diff
@@ -532,13 +540,13 @@ func (npc *NetworkPolicyController) ListNetworkPolicies(lnwname string, namespac
 	return asSelector, asPeer, nil
 }
 
-func (npc *NetworkPolicyController) insertPeerRule(policyChain, subnet string) error {
-	rulespec := []string{"-s", subnet, "-j", "ACCEPT"}
+func (npc *NetworkPolicyController) insertPeerRule(policyChain, selector, peer string) error {
```

Comment:
Here selector and peer represents subnets, so parameter names can be kept accordingly